### PR TITLE
MAT-8514 remove markdown extension- no longer maintained, need to fin…

### PR DESCRIPTION
**Pull request by adongare**
…d alternative

## MADiE PR

Jira Ticket: [MAT-8514](https://jira.cms.gov/browse/MAT-8514)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: react/components/RichTextEditor/index.jsx [hunk 1]
```diff
@@ -174,12 +175,12 @@ const RichTextEditor = ({
     label,
     onChange,
     content,
+    canEdit,
```
Comment (RohitKandimalla) [inline react/components/RichTextEditor/index.jsx:178]: Not a big deal, but in order to maintain consistency in our individual component props of this repo, May be we can rename it to "disabled" prop instead of "canEdit" ?

**Review by sb-prateekkeerthi — APPROVED** [review]

File: react/components/RichTextEditor/index.jsx (hunk 1 above)
Comment (adongare) [reply react/components/RichTextEditor/index.jsx]: consistency is good. did not pay attention to other components . updated it. thanks for caching it.
